### PR TITLE
Lint using PSScriptAnalyzer

### PR DESCRIPTION
**Pull request by Ana06**
Add [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) to out test suite.

Note I have disabled some rules (the ones that seem less important and/or difficult to fix) to avoid making a lot of changes in one PR. We can enable them later.
Screenshot 2022-01-13 at 18 45 04

Related to https://github.com/mandiant/VM-Packages/issues/6.

### CODE REVIEW

File: packages/notepadpp.vm/notepadpp.vm.nuspec [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>notepadpp.vm</id>
-    <version>8.1.9.3</version>
+    <version>8.1.9.300</version>
```
Comment (Ana06) [inline packages/notepadpp.vm/notepadpp.vm.nuspec:5]: This version was not following the documentation.

**Comment by Ana06** [comment]
05329b0  fixes the dependency problem we had. What is failing now is also failing in the daily run.

File: packages/common.vm/common.vm.nuspec [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>common.vm</id>
-    <version>0.0.0.20220113</version>
+    <version>3.0.1.20220222</version>
```
Comment (MalwareMechanic) [inline packages/common.vm/common.vm.nuspec:5]: lol can't remember, but why did we change it from a metapackage version to a non-metapackage version? Was this to make merge easier and we'd change later?

**Review by MalwareMechanic — APPROVED** [review]

File: packages/common.vm/common.vm.nuspec (hunk 2 above)
Comment (Ana06) [reply packages/common.vm/common.vm.nuspec]: typo, fixed!

**Pull request merged**